examples/search-knn.js

```diff
@@ -13,71 +13,79 @@ await client.connect();
 // Create an index...
 try {
   // Documentation: https://redis.io/docs/stack/search/reference/vectors/
-  await client.sendCommand([
-    "FT.CREATE",
-    "idx:knn-example",
-    "SCHEMA",
-    "v",
-    "VECTOR",
-    "HNSW",
-    "6",
-    "TYPE",
-    "FLOAT32",
-    "DIM",
-    "2",
-    "DISTANCE_METRIC",
-    "COSINE",
-  ]);
+  await client.ft.create('idx:knn-example', {
+    v: {
+      type: SchemaFieldTypes.VECTOR,
```

Comment:
Oh nice!!! This is exactly what I wanted to do but couldn't figure out the API for. Thank you.